.github/workflows/reusable-linters.yml

```diff
@@ -21,7 +21,7 @@ env:
   PY_COLORS: 1  # Recognized by the `py` package, dependency of `pytest`
   PYTHONIOENCODING: utf-8
   PYTHONUTF8: 1
-  PYTHON_LATEST: 3.9
+  PYTHON_LATEST: 3.12
```

Comment:
Does this need to be updated too? Pretty sure it affects things other than Sphinx.. When I talked about updating things together, I meant RTD that only runs Sphinx, not affecting other places.
At least, document somewhere in the PR what's affected for history.

Reply:
Yes, thats where the last run failed on https://github.com/aio-libs/multidict/actions/runs/10783124504/job/29904388549?pr=999

Reply:
Maybe better to keep it separate in https://github.com/aio-libs/multidict/pull/1008 for this case since it affects other areas?

Reply:
yep